CHANGELOG.md

```diff
@@ -77,9 +77,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
 ### Added
+- [SearchPipeline] Add new search pipeline processor type, SearchPhaseResultsProcessor, that can modify the result of one search phase before starting the next phase.([#7283](https://github.com/opensearch-project/OpenSearch/pull/7283))
```

Comment:
These should probably not be there

Reply:
Hmm, I didn't see that. I will remove them. Must be from some rebase.